content/en/docs/concepts/configuration/secret.md

```diff
@@ -667,8 +667,8 @@ it to read a file.
 
 When deploying applications that interact with the secrets API, access should be
 limited using [authorization policies](
-https://kubernetes.io/docs/admin/authorization/) such as [RBAC](
-https://kubernetes.io/docs/admin/authorization/rbac/).
+/docs/admin/authorization/) such as [RBAC](
+/docs/admin/authorization/rbac/).
```

Comment:
This is a good change, but if we could take it even further, that would be better. The only reason these links work is because we have redirects in place, pointing from the old location, which still appears here, to the new location. Could you please amend to put the new location here?

/docs/reference/access-authn-authz/authorization/
/docs/reference/access-authn-authz/rbac/